src/annotator/adder.tsx

```diff
@@ -1,38 +1,38 @@
 import { render } from 'preact';
 
 import AdderToolbar from './components/AdderToolbar';
+import type { Command } from './components/AdderToolbar';
 import { isTouchDevice } from '../shared/user-agent';
+import type { Destroyable } from '../types/annotator';
+
 import { createShadowRoot } from './util/shadow-root';
 
 /**
- *  @typedef {1} ArrowPointingDown
  * Show the adder above the selection with an arrow pointing down at the
  * selected text.
  */
 export const ARROW_POINTING_DOWN = 1;
 
 /**
- *  @typedef {2} ArrowPointingUp
  * Show the adder above the selection with an arrow pointing up at the
  * selected text.
  */
 export const ARROW_POINTING_UP = 2;
 
-/**
- *  @typedef {ArrowPointingDown|ArrowPointingUp} ArrowDirection
- * Show the adder above the selection with an arrow pointing up at the
- * selected text.
- */
+type ArrowPointingDown = typeof ARROW_POINTING_DOWN;
+type ArrowPointingUp = typeof ARROW_POINTING_UP;
```

Comment:
Instead of defining first the `type ArrowPointingDown = 1` and then using it in `const ARROW_POINTING_DOWN: ArrowPointingDown = 1` I'm letting TS infer the type here.

I like this approach when you have constant values because you avoid "duplicating" the definition, and the constant and its type are always in sync.

Reply:
It looks like the only consumer of the exported `const`s are this component's own tests. You could consider using an `enum` here if you like (and then you wouldn't have to create a second type).

e.g. 

```
export enum ArrowDirection {
  DOWN = 1,
  UP
};
```

Reply:
Works for me. I didn't want to do such change because I wasn't sure what was your opinion on enums. They are quite controversial.

Reply:
The `enum` is actually a much nicer solution, because it covers the 3 types (or the union type) and the constants in one go. I have replaced it.